packages/code-du-travail-frontend/src/contributions/extractMdxContentUrl.js

```diff
@@ -0,0 +1,18 @@
+// Extract external content url from Content tag markdown
+const extractMdxContentUrl = markdown => {
+  if (!markdown) return;
+  // Check Content tag exist on markdown
+  const contentRegExp = new RegExp("<s*Content[^](.*?)s*/>", "g");
+  const contentTag = markdown.match(contentRegExp);
+
+  if (contentTag && contentTag.length > 0) {
+    // Extract URL from Content tag, only one for now
+    const contentUrl = contentTag[0].match(/href="(.*?)"/g);
+    if (contentUrl.length === 0) return;
+
+    return contentUrl[0].slice(6, -1);
+  }
+  return;
+};
```

Comment:
For me `s*` in a RegExp is one or more "s" :)

This should work ?

```js
const extractMdxContentUrl = markdown => {
  if (!markdown) return;
  // Check Content tag exist on markdown
  const contentTag = markdown.match(/<Content.*?href="([^"]+)".*?>/);
  return contentTag && contentTag[1];
};
```

Reply:
It's better thanks